src/systemtest/java/util/PropertiesHandler.java

```diff
@@ -0,0 +1,28 @@
+package util;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.util.Map.Entry;
+import java.util.Properties;
+
+public class PropertiesHandler {
+
+    public static void setProperties() throws Throwable {
+        final String systemTestFilePath;
+
+        if (System.getProperty("propertiesFile") != null) {
+            systemTestFilePath = System.getProperty("propertiesFile");
```

Comment:
I guess we want specify propertiesFilePath from environment variables,, so you can have a check for that also: System.getEnvVariable("propertiesFilePath") or what the name of the fucntion name.
Is this running with Spring base classes or is it a standalone Junit/Cucumber, if it is a standalone Junit/cucumber, then you need to implement the handling of Java Properties and Environment Java Properties.

Reply:
I look into this....

Reply:
and property should be name similar to the other properties, e.g. : ei.config.properties.file.path=....   or something similar.